src/lib/tree/node.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {
+  ContentChildren,
+  Directive,
+  Input,
+  QueryList
+} from '@angular/core';
+import {
+  CdkNestedTreeNode,
+  CdkTreeNodeDef,
+  CdkTreeNode,
+} from '@angular/cdk/tree';
+import {MatTreeNodeOutlet} from './outlet';
+
+
+/** Workaround for https://github.com/angular/angular/issues/17849 */
```

Comment:
That issue is closed- is this still necessary? If so we should file a new issue.